README.md

```diff
@@ -76,44 +90,40 @@ $ semgrep -e '$X == $X' --lang=py path/to/src
 | Migrate from deprecated APIs      | [DES is deprecated](https://semgrep.dev/editor?registry=java.lang.security.audit.crypto.des-is-deprecated), [Deprecated Flask APIs](https://semgrep.dev/editor?registry=python.flask.maintainability.deprecated.deprecated-apis), [Deprecated Bokeh APIs](https://semgrep.dev/editor?registry=python.bokeh.maintainability.deprecated.deprecated_apis) |
-| Apply automatic fixes             | [Use listenAndServeTLS](https://semgrep.dev/s/clintgibler:use-listenAndServeTLS)   
-
+| Apply automatic fixes             | [Use listenAndServeTLS](https://semgrep.dev/s/clintgibler:use-listenAndServeTLS)                                                                                                                                                                                                                                                                       |
 
-### Integrations
-
-Visit [Integrations](https://semgrep.dev/docs/integrations/) to learn about Semgrep editor, commit, and CI integrations. When integrated into CI and configured to scan pull requests, Semgrep will only report issues introduced by that pull request; this lets you start using Semgrep without fixing or ignoring pre-existing issues!
+### Extensions
 
+Visit [Docs > Extensions](https://semgrep.dev/docs/extensions/) to learn about Semgrep in your editor or pre-commit. When integrated into CI and configured to scan pull requests, Semgrep will only report issues introduced by that pull request; this lets you start using Semgrep without fixing or ignoring pre-existing issues!
 
 ### Documentation
 
-Browse the full Semgrep [documentation on the website](https://semgrep.dev/docs). If you’re new to Semgrep, check out [Getting started](https://semgrep.dev/docs/getting-started/) or the [interactive tutorial](https://semgrep.dev/learn).
-
+Browse the full Semgrep [documentation on the website](https://semgrep.dev/docs). If you’re new to Semgrep, check out [Docs > Getting started](https://semgrep.dev/docs/getting-started/) or the [interactive tutorial](https://semgrep.dev/learn).
 
 ### More
 
-* [Frequently asked questions (FAQs)](https://semgrep.dev/docs/faq/)
-* [Contributing](https://semgrep.dev/docs/contributing/)
-* [Ask questions in the r2c Community Slack](https://r2c.dev/slack)
-* [CLI reference and exit codes](https://semgrep.dev/docs/cli-usage)
-* [r2c YouTube channel with Semgrep presentation videos](https://www.youtube.com/channel/UC5ahcFBorwzUTqPipFhjkWg)
-* [License (LGPL-2.1)](LICENSE)
-
+- [Frequently asked questions (FAQs)](https://semgrep.dev/docs/faq/)
+- [Contributing](https://semgrep.dev/docs/contributing/)
+- [Ask questions in the r2c Community Slack](https://r2c.dev/slack)
+- [CLI reference and exit codes](https://semgrep.dev/docs/cli-usage)
```

Comment:
```suggestion
- [Command line reference and exit codes](https://semgrep.dev/docs/cli-usage)
```

Reply:
@pabloest Note that this page is still titled CLI Reference (looked best in the nav bar of docs)